### ISSUE TITLE
New operator for aggregation of elements in a tensor

### ISSUE THREAD

**Issue by rajas1998** [flexflow/FlexFlow#479]
We require a new operator to do aggregation over elements in a tensor -

For example, consider the following pytorch code to do aggregation along an axis - (https://pytorch.org/docs/stable/generated/torch.sum.html)
```
import torch

x = torch.arange(6).reshape(3,2)

print(x)

tensor([[0, 1],
        [2, 3],
        [4, 5]])

torch.sum(x, 1)

tensor([1, 5, 9])
```

We also require an operator to compute the maximum along an axis, similar to the sum reduction (https://pytorch.org/docs/stable/generated/torch.max.html)

@jiazhihao @lockshaw

**Comment by lockshaw** [comment]
@jiazhihao Thoughts on this? It appears I assigned it to you but we can shift it around if you're short on time :slightly_smiling_face:

**Comment by jiazhihao** [comment]
#544 implements Keras' reduce_sum:https://www.tensorflow.org/api_docs/python/tf/math/reduce_sum, which should be functionally equivalent to PyTorch's sum operator.
